packages/ui/lib/components/dialog/__tests__/__snapshots__/Dialog.test.tsx.snap

```diff
@@ -82,7 +82,7 @@ exports[`Test Dialog Component > Should match the snapshot with all props 1`] =
                 >
                   <path
                     d="M0 24C0 10.7452 10.7452 0 24 0C37.2548 0 48 10.7452 48 24C48 37.2548 37.2548 48 24 48C10.7452 48 0 37.2548 0 24Z"
-                    fill="currentColor"
+                    fill="#E6F9EF"
```

Comment:
why can't we get the value to work dynamically anymore? Is there some impediment or blocking issue?